### PR TITLE
SEO config for API Reference

### PR DESCRIPTION
**Pull request by TaylorKrusen**
Testing whether metadata resolves to preview build. Can't demo in local.

### CODE REVIEW

**Comment by TaylorKrusen** [comment]
Confirmed this approach will work. Hold PR for additional commits.

File: openapi-sdk.yaml [hunk 1]
```diff
@@ -3277,112 +3417,10 @@ paths:
           source:
             $ref: examples/TeamInfo.py
       x-hideOn: doc
-  '/team/members/{team_id}':
-    get:
-      tags:
-        - Team
-      summary: 'List Team Members'
-      description: 'Provides a paginated list of members (and their roles) that belong to a given team.'
-      operationId: teamMembers
```
Comment (fenghao-chen) [inline openapi-sdk.yaml:3286]: is the deletion intentional?

Comment (TaylorKrusen) [reply openapi-sdk.yaml:3286]: Nope! Thanks for catching.

**Review by fenghao-chen — APPROVED** [review]
LGTM

**Pull request merged**